android/library/src/main/java/io/logto/android/Logto.kt

```diff
@@ -100,22 +100,18 @@ class Logto(
     private val tokenSetStorageSharedPreferencesName =
         "$STORAGE_SHAREDPREFERENCES_NAME_PREFIX::${logtoConfig.cacheKey}"
 
-    private var tokenSetCache: TokenSet? = null
-        set(value) {
-            if (value != null) {
-                field = value
-                accessTokenExpiresAt = Utils.expiresAtFromNow(value.expiresIn)
-            } else {
-                field = null
-                accessTokenExpiresAt = 0L
-            }
-        }
+    private var tokenSet: TokenSet? = null
 
     private var accessTokenExpiresAt: Long = 0L
 
-    private fun updateTokenSet(tokenSet: TokenSet?) {
-        tokenSetCache = tokenSet
-        tokenSetStorage?.tokenSet = tokenSet
+    private fun updateTokenSet(updatedTokenSet: TokenSet?) {
+        tokenSet = updatedTokenSet
+        tokenSetStorage?.tokenSet = updatedTokenSet
```

Comment:
move the `accessTokenExpiresA`t as a getter property  of `tokenSet` so we could keep the single source of truth